src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -81,7 +83,8 @@ public class OrganizationService : IOrganizationService
         ICurrentContext currentContext,
         ILogger<OrganizationService> logger,
         IProviderOrganizationRepository providerOrganizationRepository,
-        IProviderUserRepository providerUserRepository)
+        IProviderUserRepository providerUserRepository,
+        IServiceAccountRepository serviceAccountRepository)
```

Comment:
With this change, I can't start `Identity` anymore. I think it's because `Identity` depends on `OrganizationService`, but its assembly doesn't include a reference to the Bitwarden Licensed code in `Bit.Core.Infrastructure.Infrastructure.EntityFramework.SecretsManager.Repositories`, where the implementation for `ServiceAccountRepository` lives.

We don't want to add Service Account dependencies to the `Identity` assembly, so we may have to make a noop service. I've posted in `#code` to see if there's a better way, but let me know if you have any ideas here.

Reply:
We also need a solution here so that customers can still run OSS versions of the server (which needs to exclude Bitwarden Licensed code)

Reply:
I think the easiest solution would be to create `NoopServiceAccountRepository` under `Core.Repositories.Noop`.

The ideal solution would be to remove BL dependencies from `OrganizationService` but we'll have to refactor that some other time.

Reply:
I went with  NoopServiceAccountRepository option